frontend/test/__support__/cypress.js

```diff
@@ -27,13 +29,18 @@ export function restore(name = "default") {
   cy.request("POST", `/api/testing/restore/${name}`);
 }
 
-export function popover(callback) {
-  const p = cy.get(".PopoverContainer");
-  return callback ? callback(p) : p;
+// various Metabase-specific "scoping" functions like inside popover/modal/navbar/main content area
+export function popover() {
+  return cy.get(".PopoverContainer");
+}
+export function modal() {
+  return cy.get(".ModalContainer");
+}
+export function nav() {
+  return cy.get("nav");
 }
-export function modal(callback) {
-  const m = cy.get(".ModalContainer");
-  return callback ? callback(m) : m;
+export function main() {
+  return cy.get("nav").next();
```

Comment:
Is there any scenario where this might return something unexpected? I'm personally 99% sure it wouldn't, it just stuck out to me while looking through this.

Reply:
`App.jsx` looks like this:

```
        <div className="relative">
          {currentUser && <Navbar location={location} />}
          {errorPage ? getErrorComponent(errorPage) : children}
          <UndoListing />
        </div>
```

So I think the main case where it wouldn't work is if there's no navbar, but the main use case I had for this was when there was a button in both the navbar and main area with the same name.